MvvmCross/Core/Binding/Bindings/Target/MvxEventNameTargetBinding.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Reflection;
+using System.Windows.Input;
+
+namespace MvvmCross.Binding.Bindings.Target
+{
+    public class MvxEventNameTargetBinding : MvxTargetBinding
+    {
+        private readonly EventInfo _targetEventInfo;
+        private readonly bool _useEventArgsAsCommandParameter;
+
+        private ICommand _currentCommand;
+
+        public MvxEventNameTargetBinding(object target, string targetEventName, bool useEventArgsAsCommandParameter = true) : base(target)
+        {
+            _targetEventInfo = target.GetType().GetTypeInfo().GetDeclaredEvent(targetEventName);
+            _useEventArgsAsCommandParameter = useEventArgsAsCommandParameter;
+
+            //  addMethod is used because of error:
```

Comment:
According to the link, this should not happen anymore and using `AddEventHandler` should be OK to use

Reply:
With weak event subscriptions this code is not needed anymore, as all reflection is doing inside weak subscribing logic